### PR TITLE
adding link to form to join emod slack comminity

### PR DESCRIPTION
**Pull request by stitova-idm**

### CODE REVIEW

**Review by Bridenbecker — CHANGES_REQUESTED** [review]
I appreciate that how important you are making this, but I don't think it should be the first thing.  I would like to see a Community section, maybe right before Support and Contribution.  It should say something like:

```
#Community

The EMOD Community includes researchers and software developers mostly involved in malaria and HIV research.  We value mutual respect, openness and a willingness to help each other.  If this sounds like you, then we encourage you to join our EMOD Slack Community by filling out this form:

https://forms.office.com/r/sjncGvBjvZ
```

Or at least something like that

**Comment by stitova-idm** [comment]
Ideally I'd like to just put that into our "About" section in the repo. :/

I'll adjust.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,15 @@ documentation will be output to the docs/_build/html directory.
...
 
+##Community
+
+The EMOD Community is made up of researchers and software developers, primarily focused on malaria and HIV research.
+We value mutual respect, openness, and a collaborative spirit. If these values resonate with you, 
+we invite you to join our EMOD Slack Community by completing this form:
+
+https://forms.office.com/r/sjncGvBjvZ
+
+
```
Comment (Bridenbecker) [inline README.md:56]: Fantastic!!!  Much better than I wrote!!

**Review by Bridenbecker — APPROVED** [review]
Love it

File: README.md (hunk 1 above)
Comment (stitova-idm) [reply README.md:56]: Thank ChatGPT. lol.

**Pull request merged**